--- xenctrl.h	2015-11-05 10:25:36.224572668 +0100
+++ xenctrl.h	2015-11-05 10:25:39.244572726 +0100
@@ -119,7 +119,6 @@
 typedef struct xc_interface_core xc_evtchn;
 typedef struct xc_interface_core xc_gnttab;
 typedef struct xc_interface_core xc_gntshr;
-typedef enum xc_error_code xc_error_code;
 
 
 /*
@@ -1764,6 +1763,7 @@
   XC_OUT_OF_MEMORY = 4,
   /* new codes need to be added to xc_error_level_to_desc too */
 };
+typedef enum xc_error_code xc_error_code;
 
 #define XC_MAX_ERROR_MSG_LEN 1024
 typedef struct xc_error {
